MinVer/build/MinVer.targets

```
@@ -33,6 +39,7 @@
 
   <Target Name="MinVer" BeforeTargets="CoreCompile;GenerateNuspec" DependsOnTargets="MinVer_GetVersion" Condition="'$(UsingMicrosoftNETSdk)' == 'true' And '$(DesignTimeBuild)' != 'true'">
     <PropertyGroup>
+      <AssemblyFileVersion>$(MinVerMajor).0.0.0</AssemblyFileVersion>

```

The property we need to set here is `AssemblyVersion`, but if we set `AssemblyVersion`, we also need to set `FileVersion` so that it continues to be set the same way it has been before this PR.

```suggestion
      <AssemblyVersion>$(MinVerMajor).0.0.0</AssemblyVersion>
      <FileVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch).0</FileVersion>
```